pyk/src/pyk/kcfg/kcfg.py

```diff
@@ -218,7 +218,14 @@ def to_dict(self) -> dict[str, Any]:
         def from_dict(dct: dict[str, Any], nodes: Mapping[int, KCFG.Node]) -> KCFG.Edge:
             return KCFG.Edge(nodes[dct['source']], nodes[dct['target']], dct['depth'], tuple(dct['rules']))
 
-        def to_rule(self, label: str, claim: bool = False, priority: int | None = None) -> KRuleLike:
+        def to_rule(
+            self,
+            label: str,
+            claim: bool = False,
+            priority: int | None = None,
+            definition: KDefinition | None = None,
+            minimize: bool = False,
```

Comment:
If you rename the parameter in `cterm_build_rule`, this has to be updated as well.

Reply:
Done.